policy-controller/src/lib.rs

```diff
@@ -113,18 +119,23 @@ impl DiscoverOutboundPolicy<OutboundDiscoverTarget> for OutboundDiscover {
     async fn watch_outbound_policy(
         &self,
         OutboundDiscoverTarget {
-            service_name,
-            service_namespace,
-            service_port,
+            kind,
+            name,
+            namespace,
+            port,
             source_namespace,
         }: OutboundDiscoverTarget,
     ) -> Result<Option<OutboundPolicyStream>> {
-        match self.0.write().outbound_policy_rx(
-            service_name,
-            service_namespace,
-            service_port,
-            source_namespace,
-        ) {
+        if let TargetKind::UnmeshedNetwork = kind {
+            tracing::debug!("UnmeshedNetwork network is not supported yet");
```

Comment:
it might be worth promoting this (_and the event above_) to a warn-level event. i'm not strongly opinionated about this, but we should warn a user if a feature isn't supported but is possibly in use.

Reply:
I think the determining factor should be how rare and exceptional this event is. If it's possible for legitimate configurations to trigger this, is this just going to spam their logs? What do we expect them to do about it?

In this case, I think we expect to encounter this fairly commonly in situations that operators should not care about.

Reply:
More importantly this is rather temporary. We will hopefully really be removing this codepath very very soon. The purpose of this PR is to introduce a non functional change and this is why we drop this resolution on the floor. But the overall goal is that we should transition to actually... supporting it. 

I will probably add a comment to elaborate on that here.